src/components/Pagination/Pagination.stories.tsx

```diff
@@ -3,15 +3,58 @@ import { Story } from "@storybook/react/types-6-0";
 import { withDesign } from "storybook-addon-designs";
 
 import Pagination, { PaginationProps } from "./Pagination";
+import { withQuery } from "@storybook/addon-queryparams";
 
 export default {
   title: "Pagination",
   component: Pagination,
-  decorators: [withDesign],
+  decorators: [withDesign, withQuery],
 };
 
-const PaginationTemplate: Story<PaginationProps> = args => {
-  const [page, setPage] = useState(7);
+const PaginationPageHrefTemplate: Story<PaginationProps> = args => {
+  const urlParams = new URLSearchParams(document.location.search);
+  const pageNumber = Number(urlParams.get("page"));
+  const location = window.location;
+
+  const getPageHref = page => {
```

Comment:
Honestly, this is the most important part of the story re: teaching other people how to use the component.  Should I add this to a separate code block in `docs` since it doesn't show up in the "show code" block?

Reply:
Per discussion, this issue will be resolved under a separate documentation-specific ticket.